lib/Data/GoogleCloudStorage.php

```diff
@@ -100,21 +112,19 @@ private function _getKey($pasteid)
      */
     private function _upload($key, $payload)
     {
-        $metadata = array_key_exists('meta', $payload) ? $payload['meta'] : array();
```

Comment:
Please advise on this part. Removing it and using the payload for the metadata made it work again. Is there any particular reason why some of the fields are stripped?